pkgs/applications/misc/paperjam/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib , stdenv, fetchgit, qpdf, libpaper, asciidoc, }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "paperjam";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "git://git.ucw.cz/paperjam.git";
+    # by recording the exact commit, we can embed it into the binary
+    # this can't be done automatically as fetchgit removes .git directory
+    rev = "90173f436b39bfe39491aa356a218ac185618d1c";
+    hash = "sha256-F0bcYXQhtaI48ULhi7o8gLlC0jIuxPIKTq1cABrP2Iw=";
+  };
+
+  buildInputs = [
+    qpdf
+    libpaper
+    asciidoc
+  ];
+
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX=/"
+    # prevent real build date which is impure
+    "BUILD_DATE=\\<unknown\\>"
```

Comment:
Why not use Epoch, 1970-01-01?

Reply:
It is only used in the output of `paperjam --version` and I feel like it expresses the intent better. The build date has no real relation to the epoch, I could as well claim it was `sometime in the past` or `0001-01-01` all being either false or useless.

Reply:
Understood.